ci-operator/step-registry/openshift-qe/installer/azure/ipi/ovn/multi-ipsec/openshift-qe-installer-azure-ipi-ovn-multi-ipsec-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-qe-installer-azure-ipi-ovn-multi-ipsec
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+      CONTROL_PLANE_INSTANCE_TYPE: Standard_D8ps_v5
```

Comment:
let's move the BASE_DOMAIN, COMPUTE_NODE_TYPE, CONTROL_PLANE_INSTANCE_TYPE out of this file so the user can set them at the top level and wont get confused when they set these variables at the top level and it to properly take effect  

I think the same thing applies for **cluster_profile** and **OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE**. I think specifically cluster_profile could be used by dev and other profiles so dont want it hidden underneath here

Reply:
Agree with what Paige said.
And my understanding is, if we have `OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest`, it's possible to put the jobs directly into the yaml file openshift-qe-ocp-qe-perfscale-ci-main__azure-4.17-nightly-x86.yaml, instead of having a new config yaml.